src/corehost/common/trace.cpp

```diff
@@ -6,17 +6,23 @@
 #include <mutex>
 
 // g_trace_verbosity is used to encode COREHOST_TRACE and COREHOST_TRACE_VERBOSITY to selectively control output of
-//    trace::warn(), trace::info(), and trace::verbose()
+//    trace::warn(), TRACE_INFO(), and TRACE_VERBOSE()
```

Comment:
I still see `trace::warn` instead of `TRACE_WARNING` in the comment... can you please double check?

Reply:
Oops. Done.